birdhouse/scripts/create-magpie-users

```diff
@@ -98,7 +98,7 @@ if [ -z "$MAGPIE_CLI_CONF" ]; then
 fi
 
 if [ -z "$MAGPIE_CLI_IMAGE" ]; then
-    MAGPIE_CLI_IMAGE="pavics/magpie:3.8.0"
+    MAGPIE_CLI_IMAGE="pavics/magpie:3.12.0"
```

Comment:
I would suggest we move the magpie image into `default.env` and make `create-magpie-users` source `default.env`, just like it already source `env.local` https://github.com/bird-house/birdhouse-deploy/blob/1719c68a8d7dafdeb88b035dc7a9bccc72c48c57/birdhouse/scripts/create-magpie-users#L72-L75

This way there is only 1 place to update the magpie image next time.

I assume we will be updating magpie a lot more frequent so we will all benefit from avoiding this duplication.

Reply:
Good idea.